Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG PGTARGET=16
 
 # We use Alpine as a base image to compile older
 # PostgreSQL versions in, then copy the binaries
-# into the PG 15 Alpine image
+# into the PG <PGTARGET> Alpine image
```

Comment:
may want to phrase this differently, maybe

```suggestion
# into the official PG Alpine image
```

Reply:
Good thinking, that sounds decent and won't need updating for each release. :smile: